### DISCUSSION TITLE
Feature: Dupliate node (workflow)

### DISCUSSION THREAD

**Discussion by AndyMik90** [langgenius/dify#3252] [Suggestion]
When working with bigger workflows, it would be nice to be able to duplicate nodes so the settings/prompts follow into the duplicated node.

**Comment by Semonxue** [comment]
I really need it. The copy and paste always get mass.

**Comment by crazywoola** [comment]
A small tip: 

> ctrl + c & ctrl + v

Will do the trick.

**Comment by AndyMik90** [comment]
Closed as this was fixed with paste functionality of nodes.
